spec/unit/filter_option_parser_spec.rb

```diff
@@ -32,26 +32,26 @@
       it "handles simple filter definitions" do
         filter = subject.parse(
           "inspect",
-          "exclude" => "/unmanaged_files/files/name=/foo"
+          "exclude" => "/unmanaged_files/name=/foo"
         )
 
         expect(filter.to_array).to match_array([
-          "/unmanaged_files/files/name=/foo"
+          "/unmanaged_files/name=/foo"
```

Comment:
Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.